nav/ui/manual_nav_dialog.py

```
@@ -1,5 +1,5 @@
 import math
-from typing import Optional, Tuple, cast
+from typing import Any, Optional, Tuple, cast
```

Ruff (0.14.5)

`typing.Tuple` is deprecated, use `tuple` instead

(UP035)